packages/web-runtime/src/helpers/language.ts

```diff
@@ -16,3 +18,31 @@ export const setCurrentLanguage = ({
     document.documentElement.lang = currentLanguage
   }
 }
+
+/**
+ * Loads all app translations for one given language.
+ * This should be called each time the language is being changed.
+ */
+export const loadAppTranslations = ({
+  apps,
+  gettext,
+  lang
+}: {
+  apps: Record<string, ApplicationInformation>
+  gettext: Language
+  lang: string
+}) => {
+  const appTranslations = Object.values(apps).reduce<Translations>((acc, app) => {
+    const { translations } = app
+
+    if (gettext.translations[lang] && translations[lang]) {
+      acc = merge(acc, translations[lang])
+    }
+
+    return acc
+  }, {})
```

Comment:
It's my personal taste, but I always feel it's weird to use `reduce` in this way. This is basically the same as:

```
let acc: Translations = {}
Object.values(apps).forEach(app => {
    const { translations } = app
    if (gettext.translations[lang] && translations[lang]) {
      acc = merge(acc, translations[lang])
    }
})
```
isn't it?

Reply:
possibly could also be `const acc` and then just `Object.assign` instead of merge

Reply:
Yup that works as well. No hard feelings from my side 🤷